### DISCUSSION TITLE
Galaxyharvester.net website down

### DISCUSSION THREAD

**Discussion by Mullet365** [pwillworth/galaxyharvester#53] [General]
Hey @pwillworth just letting you know that the galaxyharvester.net website is currently down

**Comment by pwillworth** [comment]
Hi, yes, my hosting provider is having a service disruption right now.  Hoping they resolve it soon.

**Reply by Mullet365** [reply]
Appreciate the swift reply - Good to know it's just a simple service disrupt & not some major issue. Thank you!
